meta-cube/recipes-core/images/cube-essential_0.1.bb

```diff
@@ -42,5 +42,68 @@ INITRAMFS_IMAGE = "cube-builder-initramfs"
 # We want it separate, and not bundled with the kernel by default.
 INITRAMFS_IMAGE_BUNDLE ?= ""
 
+IMAGE_FEATURES += "read-only-rootfs"
```

Comment:
I still think we may need a way (even if only for debug purposes) to make this an optional feature. It also will test that we haven't made any assumptions that break a "writable" configuration.

So perhaps we can test on a distro flag, or simply a variable that could be added to local.conf, something like: OVERC_ESSENTIAL_MODE.

It can be "read-only" by default, but if it is set to "read-write", we shouldn't add read-only-rootfs to the IMAGE_FEATURES.